pwndbg/wrappers/readelf.py

```diff
@@ -1,29 +1,31 @@
+from enum import Enum
+from typing import Dict
+from typing import List
+
 import pwndbg.wrappers
 
 cmd_name = "readelf"
 
 
-@pwndbg.wrappers.OnlyWithCommand(cmd_name)
-def get_jmpslots():
-    local_path = pwndbg.gdblib.file.get_proc_exe_file()
-    cmd = get_jmpslots.cmd + ["--relocs", local_path]
-    readelf_out = pwndbg.wrappers.call_cmd(cmd)
-
-    return filter(_extract_jumps, readelf_out.splitlines())
+class RelocationType(Enum):
+    JUMP_SLOT = 1  # e.g.: R_X86_64_JUMP_SLOT
+    GLOB_DAT = 2  # e.g.: R_X86_64_IRELATIVE
+    IRELATIVE = 3  # e.g.: R_X86_64_GLOB_DAT
```

Comment:
Are the comments here correct? GLOB_DAT comment is on the IRELATIVE line etc.

Reply:
Any source for those flags? Can we link it?

Reply:
> Are the comments here correct? GLOB_DAT comment is on the IRELATIVE line etc.

Ahh, I messed up the copy-pasting lol. Thanks, I'll fix it.

> Any source for those flags? Can we link it?

Sure, I'll add some links for those example flags.